README.md

```diff
@@ -1,30 +1,91 @@
-![logo]
-===================
-
+<!--
+  ~ Copyright (c) 2018 Goldman Sachs and others.
+  ~ All rights reserved. This program and the accompanying materials
+  ~ are made available under the terms of the Eclipse Public License v1.0
+  ~ and Eclipse Distribution License v. 1.0 which accompany this distribution.
+  ~ The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+  ~ and the Eclipse Distribution License is available at
+  ~ http://www.eclipse.org/org/documents/edl-v10.php.
+  -->
 [![][travis img]][travis]
 [![][maven img]][maven]
 [![][release img]][release]
 [![][license-epl img]][license-epl]
 [![][license-edl img]][license-edl]
 
-Eclipse Collections is a collections framework for Java. It has optimized List, Set and Map implementations with a rich and fluent API.  The library provides additional data structures not found in the JDK like Bags, Multimaps and BiMaps.  The framework also provides primitive versions of Lists, Sets, Bags, Stacks and Maps with a rich and fluent API.  There is support for both Mutable and Immutable versions of all containers in the library.  The iteration protocol was inspired by the Smalltalk collection framework, and the collections are compatible with the Java Collection Framework types.
+![logo](artwork/eclipse-collections-logo.png) 
+
+
+## [Eclipse Collections](http://www.eclipse.org/collections)
+##### [中文](https://www.eclipse.org/collections/cn/index.html), [Deutsch](https://www.eclipse.org/collections/de/index.html), [Français](https://www.eclipse.org/collections/fr/index.html), [日本語](https://www.eclipse.org/collections/ja/index.html), [Português-Brasil](https://www.eclipse.org/collections/pt-br/index.html), [Русский](https://www.eclipse.org/collections/ru/index.html)
+Eclipse Collections is a collections library for Java with a rich, functional and fluent API. The library provides optimized List, Set and Map implementations and has data structures not found in the JDK including Bags, Multimaps and BiMaps. There is support for primitive versions of Lists, Sets, Bags, Stacks and Maps for all primitive types. There are Mutable and Immutable versions of all container types. The library has support for eager, lazy, and parallel iteration patterns, with eager APIs available directly on the containers. The iteration protocol was inspired by the Smalltalk collection framework, and the collections are compatible with the Java Collection Framework types.
```

Comment:
In the interest of conciseness, we can remove feature lists and put them later.

Reply:
Updated